DOCS-261 - Rework cloud soar (#4392)
SumoLogic/sumologic-documentation: docs/cloud-soar/legacy/legacy-cloud-soar-automation.md

This article only applies to organizations having a legacy Cloud SOAR instance URL matching the pattern `*.soar.sumologic.com`. If it doesn't, refer to [Cloud SOAR Automation](/docs/cloud-soar/automation/) for documentation of our latest Cloud SOAR SaaS version.
:::

## Report template

**Report Templates** allow users to build their own reports by selecting various components of an incident they wish to include in the report. These components can include incident details, evidence, hosts, observables and many others.

## Credential Manager - CyberArk Configuration

You can use CyberArk Credential Manager to manage data that will be used in integration resources.

<img src={useBaseUrl('img/cloud-soar/cyberArk1.png')} alt="Integrations" style={{border: '1px solid gray'}} width="600"/>

Using the cogwheel icon on the right in the integrations section, the main section of the CyberArk configuration opens.

<img src={useBaseUrl('img/cloud-soar/CyberArk2.png')} alt="CyberArk configuration" style={{border: '1px solid gray'}} width="400"/>

> Here you can set URL and port of the Components server, and the credentials needed to connect to CyberArk. The Enable checkbox can be enabled or disabled later.

Reviewer: ```suggestion
Here you can set URL and port of the Components server, and the credentials needed to connect to CyberArk. The **Enable** checkbox can be enabled or disabled later.
```

If enabled, when you go to open the detail of a integration resource you'll find a new checkbox (**Use CyberArk fields**) at the top already active. If the checkbox on above window is disabled, the checkbox in the resource window will be disabled by default, and it will not be possible to activate it.

<img src={useBaseUrl('img/cloud-soar/CyberArk3.png')} alt="Enable CyberArk fields" style={{border: '1px solid gray'}} width="800"/>

If the checkbox **Use CyberArk fields** is enabled, two new mandatory fields will appear:
* **Account Name** > userName in CyberArk
* **Platform ID** > platformId in CyberArk

> Near to the fields there will be the relative toggle that will enable the related field for use on CyberArk.

Reviewer: ```suggestion
Near to the fields, there will be the relative toggle that will enable the related field for use on CyberArk.
```

<img src={useBaseUrl('img/cloud-soar/CyberArk5.png')} alt="CyberArk fields enabled" style={{border: '1px solid gray'}} width="400"/>

> In the image above, you can see two custom fields of the resource with their toggles. The first field has been enabled to use CyberArk, while the second not.

Reviewer: ```suggestion
In the image above, you can see two custom fields of the resource with their toggles. The first field has been enabled to use CyberArk, while the second one hasn't.
```
